assets/scss/modules/_forms.scss

```diff
@@ -0,0 +1,391 @@
+          content: '';
+          position: absolute;
+          left: 5px;
+          top: 1px;
+          width: 5px;
+          height: 10px;
+          border: solid $dp-color-white;
+          border-width: 0 3px 3px 0;
+          transform: rotate(45deg);
+        }
+      }
+
+      label {
+        font-weight: normal;
+      }
+
+      &.error {
+        .checkmark {
+          border-color: $dp-color-red;
+        }
+      }
+    }
+  }
+
+  input:not(.dp-button):not([type='checkbox']),
+  textarea {
+    height: 40px;
+    min-width: 370px;
+    border: 1px solid $dp-color-grey;
+    border-radius: 3px;
+    background-color: $dp-color-white;
+    font-size: $dp-font-size-p;
+    line-height: 20px;
+    padding: 10px 12px;
+    margin-top: 5px;
+
+    &::placeholder {
+      color: #838370;
+      font-style: italic;
+    }
+
+    &:required {
+      box-shadow: none;
+    }
+
+    &:focus {
+      color: #525845;
+      border: 1px solid $dp-color-yellow;
+      border-radius: 3px;
+      -webkit-box-shadow: 0 0 0 2px rgba(0,0,0,0.1);
+      box-shadow: 0 0 0 2px rgba(0,0,0,0.1);
+      outline: none;
+
+      &::placeholder {
+        font-style: normal;
+      }
+    }
+  }
+
+  textarea {
+    height: 120px;
+    resize: none;
+  }
+
+  input[type="tel"].phone {
+    -moz-appearance: textfield;
+
+    &::-webkit-inner-spin-button,
+    &::-webkit-outer-spin-button {
+      -webkit-appearance: none;
+      margin: 0;
+    }
+  }
+
+  input.phone-number {
+    padding-right: 52px;
+  }
+
+  .loader {
+    position: relative;
+
+    img {
+      display: none;
+      height: 20px;
+      width: 20px;
+      position: absolute;
+      top: calc(50% - 10px);
+      left: calc(50% - 10px);
+      animation: spin 2s linear infinite;
+    }
+  }
+
+  .success-message {
+    text-align: right;
+    display: none;
+
+    p {
+      color: $dp-color-green;
+    }
+
+    .icon-success {
+      height: 35px;
+      width: 35px;
+      background: $dp-color-green;
+      color: $dp-color-white;
+      border-radius: 100%;
+      margin-left: 10px;
+      position: relative;
+      font-size: 10px;
+
+      &:before {
+        position: absolute;
+        top: 50%;
+        left: 11px;
+        transform: translateY(-50%);
+      }
+    }
+  }
+
+  .error {
+    input:not(.button):not([type='checkbox']),
+    textarea {
+      border: 1px solid $dp-color-red;
+
+      &:focus {
+        border: 1px solid $dp-color-red;
+        -webkit-box-shadow: 0 0 0 2px rgba(0,0,0,0.1);
+        box-shadow: 0 0 0 2px rgba(0,0,0,0.1);
+      }
+    }
+
+    &:after {
+      left: 0;
+      top: 100%;
+      color: $dp-color-red;
+      content: attr(data-error);
+      font-size: $dp-font-size-p;
+      line-height: 20px;
+      padding: 0;
+      position: absolute;
+      text-transform: initial;
+      white-space: nowrap;
+      z-index: 1;
+    }
+  }
+
+  &.disabled {
+    input:not(.button),
+    textarea {
+      background: #E7E5DC;
```

Comment:
This also could be in a var `#E7E5DC` and `#838370`

Reply:
use variables :)